modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/test/clazz/group/FunctionalBatchTestClassGroup.java

```diff
@@ -148,6 +148,50 @@ protected FunctionalBatchTestClassGroup(
 		setSegmentTestClassGroups();
 	}
 
+	protected String getTestBaseDirName() {
+		Job job = getJob();
+
+		if (job instanceof PluginsGitRepositoryJob) {
+			PluginsGitRepositoryJob pluginsGitRepositoryJob =
+				(PluginsGitRepositoryJob)job;
+
+			File pluginTestBaseDir =
+				pluginsGitRepositoryJob.getPluginTestBaseDir();
+
+			if ((pluginTestBaseDir != null) && pluginTestBaseDir.exists()) {
+				return JenkinsResultsParserUtil.getCanonicalPath(
+					pluginTestBaseDir);
+			}
+		}
+
+		String testBaseDirName = System.getenv("test.base.dir.name");
+
+		if ((testBaseDirName == null) || !testBaseDirName.isEmpty()) {
+			testBaseDirName = System.getenv("env.TEST_BASE_DIR_NAME");
+		}
+
+		if ((testBaseDirName == null) || !testBaseDirName.isEmpty()) {
+			testBaseDirName = System.getenv("TEST_BASE_DIR_NAME");
+		}
+
+		if ((testBaseDirName == null) || !testBaseDirName.isEmpty()) {
+			return null;
+		}
```

Comment:
@michaelhashimoto This looks strange to me.. should these conditionals be checking for testBaseDirName.isEmpty() instead of !testBaseDirName.isEmpty()?

Reply:
yeah, that's right.. nice catch.